lib/src/scss-shared/_nx-input-helpers.scss

```diff
@@ -3,19 +3,16 @@
 }
 
 @mixin focusable-toggle-input($focus-border-selector) {
-  position: relative;
-
   .nx-radio__input, .nx-checkbox__input {
-    position: absolute;
-    z-index: -1;
-
     &:focus {
       // less-nice IE11 focus style because IE11 doesn't support :focus-within
+      height: 100%;
```

Comment:
Doesn't seem to be necessary

Reply:
> Doesn't seem to be necessary

Removed: https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/74308db5d3ddf7206ab167c431feba89dc373d74